installer/helm/chart/volcano/templates/controllers.yaml

```diff
@@ -74,13 +74,13 @@ rules:
     verbs: ["get", "create", "delete"]
   - apiGroups: ["apps"]
     resources: ["daemonsets", "statefulsets"]
-    verbs: ["get"]
+    verbs: ["get", "list"]
```

Comment:
If we change all these APIs to list/watch them, I think the verbs here should be ["list","watch"]?